docs/current.settings.yaml

```diff
@@ -1,4 +1,4 @@
-config_version: 2
+config_version: 000
```

Comment:
heh, it's just us, so we could probably just say we're pre-release, choose one and just leave it